proper-nouns.csv

```diff
@@ -0,0 +1,7 @@
+Lemma,Sense,Word Class,Year,Quotation,Source,Note
+阿比西尼,/,proper noun,1857,?,A. Wylie『六合叢談』,
+阿非利加,/,proper noun,1848,?,徐繼畬『瀛環志略』,
+阿非利加州,/,proper noun,1855,阿非利加州,B. Hobson『博物新編』,
+阿弗利加,/,proper noun,1857,?,A. Wylie『六合叢談』,
+埃及,/,proper noun,1607,非洲東北部的一個國家,『幾何原本』,
+北極星,1,proper noun,1815,"A constellation, 北極星","R. Morrison, _A Dictionary of the Chinese Language in three parts_",
```

Comment:
```suggestion
```
I see I wasn’t being very clear about what it is that’s low priority: it’s a subset of proper nouns, specifically the loanwords. Which are particularly common in A.

Let’s get everything else into the main table.